macOS/DuckDuckGo/Menus/MainMenu.swift

```diff
@@ -495,6 +502,16 @@ final class MainMenu: NSMenu {
         updateAutofillDebugScriptMenuItem()
         updateShowToolbarsOnFullScreenMenuItem()
         updateWatchdogMenuItem()
+        updateWebExtensionsMenuItem()
+    }
+
+    private func updateWebExtensionsMenuItem() {
+        if let webExtensionsMenuItem,
+           webExtensionsMenuItem.parent == nil,
+           let debugMenuItem = items.first(where: { item in item.title == Self.debugMenuTitle }),
+           let debugSubmenu = debugMenuItem.submenu {
+            debugSubmenu.insertItem(webExtensionsMenuItem, at: max(0, debugSubmenu.items.count - 3))
```

Comment:
I'd rather just insert it at the last position so this is more future proof... doing `debugSubmenu.items.count - 3` won't cause trouble now but would crash if the count is ever lower than 3 (it's circumstantial).